src/utils/register-foreign-multiple.coffee

```diff
@@ -0,0 +1,65 @@
+program = require 'commander'
+msgflo_nodejs = require 'msgflo-nodejs'
+fs = require 'fs'
+path = require 'path'
+foreigner = require '../foreign-participant'
+common = require '../common'
+yaml = require 'js-yaml'
+Promise = require 'bluebird'
+
+onError = (err) ->
+  console.log err
+  process.exit 1
+
+onComplete = ->
+  process.exit 0
+
+collectMap = (val, map) ->
+  [role, definition] = val.split ':'
+  map[role] = definition
+  return map
+
+main = ->
+  program
+    .option('--broker <uri>', 'Broker address', String, '')
+    .option('--roles <role:definition>', 'Map of roles and definition files', collectMap, {})
+    .option('--interval <SECONDS>', 'How often to send discovery message', Number, 60)
+    .option('--forever <true>', 'Keep running forever', Boolean, false)
```

Comment:
`--forever` should not be an option I think, but instead always on. We generally require messages to be sent periodically now (#30)